README.md

```diff
@@ -97,19 +116,31 @@ try {
 }
 ```
 
+// Same as above, comments and noise (try/catch)
+
 As you can see from the highlighted lines above in order to enable TLS 1.3 in JDK 11 we need to specify the appropriate constant representing the protocol version over the server and client SSL sockets created and also set the appropriate cypher suites that TLS 1.3 expects again on both the client and server SSL sockets.
 
+// Please specify the lines explicitly, or use comments in the code
+
 ## Enhancing the Security Sandbox Model
 
 Now let’s assume that our TLS 1.3 server is implemented as a set of distinct Jigsaw modules (Jigsaw brings a new module system to the JDK ecosystem as of JDK 9). Moreover our server application provides a plug-in system that uses a distinct classloader to load different applications providing integrations with third-party systems. This would imply that we need a form of access control for the applications managed by our server application. The Java security sandbox comes to the rescue. It remains the same as of JDK 9 where Jigsaw modules bring subtle changes to the JDK so that the permission model can be applied to modules. This is achieved by simply introducing a new scheme (jrt) for referring to Jigsaw modules when specifying permissions in the security.policy file. For example if we install a security manager in the JSSE server we created by adding the following at the beginning of the source code:
 ```java
 System.setSecurityManager(new SecurityManager());
 ```
+// I must push back strongly
+// You should never ever set the security manager in the code
+// As you might already have been compromised
```

Comment:
That can be the the case whereby one provides a modified version of the JAR and no MAC check is done ? The other drawback I see is due the fact that in case the security manager must be switched then a recompilation needs to happen. If these are not concerns then it should be OK to set it programatically ... I will modify it though.